tests/Driver/CoreDriverTest.php

```diff
@@ -22,9 +22,15 @@ public function testNoExtraMethods()
 
     public function testCreateNodeElements()
     {
+        if(\PHPUnit\Runner\Version::id() >= 10) {
```

Comment:
this should be removed from this PR as we are *not* using phpunit 10 in our setup (intentionally).

Reply:
i removed it now